config/settings.yml

```diff
@@ -19,3 +19,7 @@ h2:
 
 # feature flag
 allow_sdr_content_changes: true
+
+# mappings from authorization role to workgroup name
+authorization_workgroup_names:
+  application_user: 'dlss:h2-app-users'
```

Comment:
After mid-sprint planning discussion today, this approach is going to change. See the issue for some updated text. Happy to chat and/or pair tomorrow. cc: @jcoyne